cmd/proxy/main.go

```diff
@@ -132,7 +193,7 @@ func main() {
 			log.Infof("Shutting down %s server", name)
 			if err := server.Shutdown(context.Background()); err != nil {
 				// Error from closing listeners, or context timeout:
-				log.Warn("HTTP server Shutdown Error", zap.Error(err))
+				Exit(errors.Wrap(err, "HTTP server Shutdown Error"))
```

Comment:
Revert

Reply:
Can you explain why? Why not report this to sentry.

Reply:
reporting is fine, but that's not what's happening. before it was a warning, not it's logging an error and exiting